docs/how-to/version-statuses.md

```diff
@@ -0,0 +1,30 @@
+
+
+## Route Information
+Navigate to: \
+```/org/{:organization}/{:project}/{:application}/versions```
+
+## Verification Steps
+Locate the Version List Table. Version statuses are displayed in the Status column.
+
+### Updating Version Status
+Click the Actions button to open the dropdown menu.
+Available actions:
+ - Set Version as Deprecated
+ - Set Version as Unavailable
+ - Set Version as Available
+ - Deploy Version
+
+## Validation Rules & Expected Behavior
+#### Deploying a Deprecated Version
+You can deploy deprecated version, but required warning should appear in the deploy modal. You must check a confirmation box acknowledging the risks before proceeding.
```

Comment:
This would be a great place to show the modal, and explain why we have it, this is written as a requirement rather than an explanation.